### PR TITLE
Update "snipers" to require weilding

### PR DESCRIPTION
**Pull request by BlitzTheSquishy**
## About the PR

gave items in the "sniper" category wielding requirements (except flintlock)

## Why / Balance

Try firing a bolt-action with one hand to the efficiency of a spess man, you cant. Also shotguns already require wielding. So the Hristov definitely should given a bullet generally has more powder than behind it than shot

## Requirements

- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: You now have to hold a rifle with TWO hands to operate it.

### CODE REVIEW

**Comment by Lyndomen** [comment]
s-s-should this be upstream :3

**Comment by BlitzTheSquishy** [comment]
> s-s-should this be upstream :3

2 changes here are for us specifically so not *entirely*

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 1]
```diff
@@ -63,6 +64,8 @@
       - CartridgeAntiMateriel
     capacity: 5
     proto: CartridgeAntiMateriel
+  - type: GunRequiresWield # Delta-V | Firing an antimateriel rifle is.. incredibly unweildy with one hand.
+
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml]: erroneous newline

**Review by deltanedas — CHANGES_REQUESTED** [review]
comments should use - not |

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 2]
```diff
@@ -46,6 +46,7 @@
   components:
   - type: Sprite
     sprite: Objects/Weapons/Guns/Snipers/bolt_gun_wood.rsi
+  - type: GunRequiresWield # Delta-V - Firing a bolt action is.. incredibly unweildy with one hand.
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml]: just `DeltaV`, consistency, see contribution guidelines

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 3]
```diff
@@ -63,6 +64,7 @@
       - CartridgeAntiMateriel
     capacity: 5
     proto: CartridgeAntiMateriel
+  - type: GunRequiresWield # Delta-V - Firing an antimateriel rifle is.. incredibly unweildy with one hand.
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml]: above

File: Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml [hunk 4]
```diff
@@ -101,6 +103,7 @@
     animation: WeaponArcThrust
     soundHit:
       path: /Audio/Weapons/bladeslice.ogg
+  - type: GunRequiresWield # Delta-V - Firing a muzzle loader is.. incredibly unweildy with one hand.
```
Comment (MilonPL) [inline Resources/Prototypes/Entities/Objects/Weapons/Guns/Snipers/snipers.yml]: above

**Review by MilonPL — DISMISSED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**